modules/openshift-cluster-maximums.adoc

```diff
@@ -5,97 +5,77 @@
 [id="cluster-maximums_{context}"]
 = {product-title} tested cluster maximums
 
-[options="header",cols="7*"]
+The values in the following table were tested independently of each other and represent the maximum for that particular resource type. It might not be valid to consider these maximums in combinations. Appropriate capacity planning and testing should be performed for large environments.
```

Comment:
We might want to mention that there are other objects created during some the test runs but not close to the maximums. For example, when we test namespaces per cluster, deployments per namespace or number of nodes, we create thousands of pods, services, deployments, secrets, configmaps, builds etc. but are not close to the cluster maximums. Thoughts?

Reply:
@chaitanyaenr Sounds good to me! Updated.